mssqlcli/mssqlcompleter.py

```diff
@@ -429,6 +433,8 @@ def case(self, word):
         return self.casing.get(word, word)
 
     def get_completions(self, document, complete_event, smart_completion=None):
+        # pylint: disable=arguments-differ
```

Comment:
nitpick: do we need the newline after this?

Reply:
Just my opinion, I don't think it hurts. I think it helps group the comment with the function definition.